Docs/user-guides/reference/02-working-with-openl-tables/03-table-types/02-decision-table/09-simple-lookup-tables.md

```
@@ -0,0 +1,14 @@
+###### Simple Lookup Table
+
+A lookup decision table with simple conditions that check equality of an input parameter and a condition value and a simple return can be easily represented as **simple lookup table**. This table is similar to simple rules table but has horizontal conditions. The number of parameters to be associated with horizontal conditions is determined by the height of the first column title cell.
+
+The simple lookup table header format is as follows:
+
+`SimpleLookup <Return type> RuleName(<Parameter type 1> parameterName1, (<Parameter type 2> parameterName2,….)`
+
+The following is an example of a simple lookup table.
+
+![](../../../../ref_guide_images/7c9fa65d2859ca54b256467063d0a345.png)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)